tests/test_runs/test_run_functions.py

```diff
@@ -55,6 +56,24 @@ class TestRun(TestBase):
         [1, 2, 7, 10, 13, 14],
     )
 
+    # if task IDs are deleted during test server maintenance, these meta data should still allow
+    # unit tests to pass by uploading a similar task at runtime
+    TASK_META_DATA = {
+        1605: {
+            "task_type": "Supervised Regression",
+            "dataset_id": 123,
```

Comment:
Could you please add that this is the dataset quake? Also, is this the same information as in the change above? If yes, can they be merged?